src/content/docs/identitymodel/endpoints/introspection.mdx

```diff
@@ -0,0 +1,108 @@
+The `TokenIntrospectionResponse` class exposes the raw response through its `Raw` property,
+and to the parsed JSON document through its `Json` property.
+In addition, it provides access to the following standard response parameters:
+
+| Property     | Value                                                                                                                                                                               |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Scopes`     | The list of scopes associated to the token or an empty array if no `scope` claim is present.                                                                                        |
+| `ClientId`   | The client identifier for the OAuth 2.0 client that requested the token or `null` if the `client_id` claim is missing.                                                              |
+| `UserName`   | The human-readable identifier for the resource owner who authorized the token or `null` if the `username` claim is missing.                                                         |
+| `TokenType`  | The type of the token as defined in [section 5.1 of OAuth 2.0 (RFC6749)](https://datatracker.ietf.org/doc/html/rfc6749#section-5.1) or `null` if the `token_type` claim is missing. |
+| `Expiration` | The expiration time of the token or `null` if the `exp` claim is missing.                                                                                                           |
+| `IssuedAt`   | The issuance time of the token or `null` if the `iat` claim is missing.                                                                                                             |
+| `NotBefore`  | The validity start time of the token or `null` if the `nbf` claim is missing.                                                                                                       |
+| `Subject`    | The subject of the token or `null` if the `sub` claim is missing.                                                                                                                   |
+| `Audiences`  | The service-specific list of string identifiers representing the intended audience for the token or an empty array if no `aud` claim is present.                                    |
+| `Issuer`     | The string representing the issuer of the token or `null` if the `iss` claim is missing.                                                                                            |
+| `JwtId`      | The string identifier for the token or `null` if the `jti` claim is missing.                                                                                                        |
+
+## JWT Response Validation :badge[v7.1]
+
+By default, **no** validation is performed on the incoming JWT response, it is only checked for valid JWT formatting.
```

Comment:
This sounds scary, but it is actually fine - just important for the user to understand. We don't do validation because we don't want a dependency on a jwt library, and because most applications don't benefit from additional checks at run time - again the value is in the non-repudiation if you saved the introspection jwt.